Doc/c-api/init.rst

```diff
@@ -54,12 +54,15 @@ The following functions can be safely called before Python is initialized:
   * :c:func:`PyMem_RawCalloc`
   * :c:func:`PyMem_RawFree`
 
-.. note::
-
-   The following functions **should not be called** before
-   :c:func:`Py_Initialize`: :c:func:`Py_EncodeLocale`, :c:func:`Py_GetPath`,
-   :c:func:`Py_GetPrefix`, :c:func:`Py_GetExecPrefix` and
-   :c:func:`Py_GetProgramFullPath` and :c:func:`Py_GetPythonHome`.
+On the contrary, the following functions **should not be called** before
+:c:func:`Py_Initialize`:
+
+* :c:func:`Py_EncodeLocale`
```

Comment:
I'm not sure this would look better. The list takes more space and attracts more attention (and distract it from the above lists). This list is less important than above lists. Actually the list of functions that should not be called before Py_Initialize() is much longer. ;-) Here are just functions that look similar to functions in the above lists.

Reply:
But removing the "note" directive LGTM.

Reply:
Maybe we need a distinct section for this list?